app/Domain/Experience/Commands/RegisterPullRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Domain\Experience\Commands;
+
+use App\Domain\Experience\ExperienceAggregateRoot;
+use App\Models\User;
+use App\Support\Uuid\Uuid;
+use Spatie\EventSourcing\Commands\AggregateUuid;
+use Spatie\EventSourcing\Commands\HandledBy;
+
+#[HandledBy(ExperienceAggregateRoot::class)]
+class RegisterPullRequest
+{
+    public static function forUser(User $user, string $reference): self
+    {
+        if (! $user->uuid) {
```

Comment:
I'll add `$user->resolveUuid()` still, and properly use it throughout all commands. 

(sidenote: the UUID is necessary because of the experience AR, so we need to ensure it exists beforehand)